jimp.d.ts

```diff
@@ -234,369 +234,266 @@ declare namespace Jimp {
         toString(): string;
         getMIME(): string;
         getExtension(): string;
-        write(path: string, cb?: Jimp.ImageCallback): this;
+        write(path: string, cb: Jimp.ImageCallback): this;
         writeAsync(path: string): Promise<Jimp>;
-        deflateLevel(l: number): this;
-        deflateLevel<T>(l: number, cb: Jimp.ImageCallback<T>): T;
-        deflateStrategy(s: number): this;
-        deflateStrategy<T>(s: number, cb: Jimp.ImageCallback<T>): T;
-        filterType(f: number): this;
-        filterType<T>(f: number, cb: Jimp.ImageCallback<T>): T;
-        rgba(bool: boolean): this;
-        rgba<T>(bool: boolean, cb: Jimp.ImageCallback<T>): T;
-        quality(n: number): this;
-        quality<T>(n: number, cb: Jimp.ImageCallback<T>): T;
-        getBase64(mime: string, cb?: GenericCallback<string, any, this>): this;
+        deflateLevel(l: number, cb?: Jimp.ImageCallback<T>): this;

```

Comment:
You have to also remove the `<T>` part in the ImageCallback, since the type parameter is not defined anywhere